bijection-hbase/src/main/scala/com/twitter/bijection/hbase/HBaseInjections.scala

```diff
@@ -58,53 +59,45 @@ object HBaseInjections {
   implicit lazy val string2BytesWritableInj =
     new ImmutableBytesWritableInjection[String] {
       override def invert(b: ImmutableBytesWritable) =
-        Try {
-          val str = Bytes.toString(b.get, b.getOffset, b.getLength)
-          if (str == null) sys.error("Invalid string")
-          else str
-        }
+        fastAttempt(b)(Bytes.toString(b.get, b.getOffset, b.getLength) match {
+          case null => sys.error("Could not read string from: $b")
```

Comment:
you need `s"Could not read ...` not the `s` to do interpolation. This is a common bug and a bit of a pain with interpolation.

I'd recommend actually: `s"$b decoded to null, which is disallowed."`